Cargo.toml

```diff
@@ -153,8 +153,8 @@ webpki-roots = "0.25"
 webrender = { git = "https://github.com/servo/webrender", branch = "0.65", features = ["capture"] }
 webrender_api = { git = "https://github.com/servo/webrender", branch = "0.65" }
 webrender_traits = { path = "components/shared/webrender" }
-webxr = { git = "https://github.com/servo/webxr" }
-webxr-api = { git = "https://github.com/servo/webxr" }
+webxr = { git = "https://github.com/wusyong/webxr", branch = "surfman" }
+webxr-api = { git = "https://github.com/wusyong/webxr", branch = "surfman" }
```

Comment:
This will need to wait for https://github.com/servo/webxr/pull/256. The lint error is because of this of course.
I want to make sure the check CI can be built first.